notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue

```diff
@@ -809,13 +807,16 @@ export default {
       const contentChildren = docElement.getElementsByTagName('body')[0].children;
       const links = docElement.getElementsByTagName('a');
       const tables = docElement.getElementsByTagName('table');
+      docElement.querySelectorAll('oembed').forEach((oembed) => {
+        oembed.innerHTML = oembed.dataset.iframe;
+        delete oembed.dataset.iframe;
+      });
```

Comment:
Sorry, I didn't get why we still need to change the saved HTML to make the display works. Can't we get a complete HTML from REST endpoint and display it as is ?
We will have a whole **CMS set of features** on top of this work. We should take the time to improve the performances of contents rendering now, to not have to make heavy and a substantial work for data migration just after few months. If we don't optimize the maximum of things now, we will probably never prioritize the performance improvements later when we will see that the page loading takes a lot of time when it's full of **SNVs**.

Reply:
Same as codeSnippets plugin, the cke media plugin needs a processing as well, because when we insert a link using the embed media plugin, in edit mode  all is OK, but when call `editor.getData()` we have only "`<oembed>usedLink</oembed>`"

Reply:
I have had to do something about this : https://github.com/Meeds-io/notes/pull/843/commits/a71cb13e8f8313ba82d6e6d17e232759bfdf8ba7#diff-43927e45889ed109fe9f553ea38b02993f822a8fb1827fa9d26c660fa5cf3c79R811-R852

`restoreEmbedded()` and `restoreunHighlited`() are needed to make the content compatible again with ckeditor in edit mode and recognized by the corresponding plugins

Reply:
ok, no need to do more, we have to re-architecture the whole stuff to enhance this, by adding by example an additional field in DB for `Content To Edit` & keep the current one for `Content To View`. This would be the good integration to make with CKEditor advanced plugins that was added by current MIP... we will need to rework this but may be in the next MIP. Let's just keep it a simple implementation for now... Thanks.